plugins/backstage-plugin-flux/package.json

```diff
@@ -46,7 +46,7 @@
     "luxon": "^3.3.0",
     "react-use": "^17.4.0",
     "styled-components": "^6.1.1",
-    "use-deep-compare": "^1.1.0",
+    "use-deep-compare": "^1.2.0",
```

Comment:
There now seems to be a 1.2.1 which may let us drop the typedefs. (not sure)

Reply:
Good stuff, that has indeed fixed it :)